### PR TITLE
Migrate to androidx preference

### PR DESCRIPTION
**Pull request by Jean-BaptisteC**
I have removed deprecations about preference manager by using android x preference library.
I have followed this [link ](https://stackoverflow.com/questions/56833657/preferencemanager-getdefaultsharedpreferences-deprecated-in-android-q)to doing the migration.

### CODE REVIEW

File: app/build.gradle.kts [hunk 1]
```diff
@@ -90,4 +90,6 @@ dependencies {
     implementation("com.afollestad:aesthetic:0.7.2")
     implementation("me.jfenn:timedatepickers:0.0.6")
     implementation("me.jfenn:AndroidUtils:0.0.5")
+
+    implementation ("androidx.preference:preference-ktx:1.2.1")
```
Comment (meenbeese) [inline app/build.gradle.kts:94]: Move this up below `core`

**Review by meenbeese — COMMENTED** [review]
Will the previous shared preferences be saved after this change?

**Comment by Jean-BaptisteC** [comment]
I have tested with an apk using android preference and I have update apk with androidx preference. My custom settings has been keep.

**Comment by meenbeese** [comment]
Ok, LGTM.

**Pull request merged**